Twitch.Net/TwitchApi.Streams.cs

```diff
@@ -74,11 +81,56 @@ public async Task<HelixPaginatedResponse<HelixStream>> GetStreams(string[] userI
             {
                 parameters.Add(new KeyValuePair<string, string>("before", before));
             }
-
+		
+	    if (string.IsNullOrEmpty(before) == false)
+            {
+                parameters.Add(new KeyValuePair<string, string>("before", before));
+            }
+		
             var responseStream = await httpClient.GetAsync(_getStreamsEndpoint, parameters);
 
             return await JsonSerializer.DeserializeAsync<HelixPaginatedResponse<HelixStream>>(responseStream);
         }
+        
+        /// <summary>
+        /// Get streams for specific usernames
+        /// </summary>
+        /// <param name="userLogins">Array of user login names. Limit: 100</param>
+        /// <param name="first">Amount of streams. Limit: 100</param>
+        /// <param name="language">Stream language. Limit: 100</param>
+        /// <param name="after">Cursor for pagination</param>
+        /// <param name="before">Cursor for pagination</param>
+        /// <returns><see cref="HelixPaginatedResponse{HelixStream}"/> with streams</returns>
+        public async Task<HelixPaginatedResponse<HelixStream>> GetStreams(string[] userLogins, int first = 20, string language = null, string after = null, string before = null)
+        {
+            using var httpClient = GetHttpClient();
+
+            var parameters = new List<KeyValuePair<string, string>>
+            {
+                new KeyValuePair<string, string>("first", first.ToString())
+            };
+
+            parameters.AddRange(userLogins.Select(userLogin => new KeyValuePair<string, string>("user_login", userLogin)));
+
+            if (string.IsNullOrEmpty(after) == false)
+            {
+                parameters.Add(new KeyValuePair<string, string>("after", after));
+            }
+
+            if (string.IsNullOrEmpty(before) == false)
+            {
+                parameters.Add(new KeyValuePair<string, string>("before", before));
+            }
 
+            if (string.IsNullOrEmpty(language) == false)
+            {
+                parameters.Add(new KeyValuePair<string, string>("language", language));
+            }
```

Comment:
Wie in einem anderen Kommentar schon beschrieben, sollte `languages` vom Typ ```string[]``` sein, weshalb hier ein ähnliches Verfahren wie in Zeile 113 genutzt werden sollte. (Bei den anderen Methoden natürlich auch)